### PR TITLE
Configure Email Service: Add credentials to yaml

### PR DESCRIPTION
**Pull request by enyia21**
#184899335

### CODE REVIEW

File: sample.env [hunk 1]
```diff
@@ -44,3 +44,10 @@ AZURE_SECRET_KEY=
 SMARTY_AUTH_ID=
 SMARTY_AUTH_TOKEN=
 
+#######################################################################################
+# Mailgun Configuration
+#######################################################################################
+# Mailgun is the default email client for form-flow-builder.   You can pass to use the
+# Mailgun api or smtp by adding a Mailgun id and key into your .env file:
```
Comment (bseeger) [inline sample.env:51]: I think we are going to use the API for MailGun, so I think you can take out the SMTP part.

**Comment by bseeger** [comment]
This looks good for the starter app repo. I think some docs need to be added to the form flow library itself for configuration of this feature.

File: sample.env (hunk 1 above)
Comment (enyia21) [reply sample.env]: I removed SMTP text from the sample.env file

**Review by bseeger — APPROVED** [review]
LGTM

**Pull request merged**